src/Spice86.Core/Emulator/Function/Dump/RecorderDataWriter.cs

```diff
@@ -60,18 +60,7 @@ public class RecorderDataWriter : RecordedDataIoHandler {
     /// <param name="suffix">The suffix to add to the file name.</param>
     public void DumpMemory(string suffix) {
         string path = GenerateDumpFileName($"MemoryDump{suffix}.bin");
-        File.WriteAllBytes(path, GenerateRelevantRam());
-    }
-
-    /// <summary>
```

Comment:
Maybe keep it until the new Generator is available ?

We can mark it [Obsolete("reason here")]